schemas/blocks/introduction.json

```diff
@@ -6,12 +6,6 @@
       "id": {
         "$ref": "../common_definitions.json#/id"
       },
-      "title": {
-        "$ref": "../string_interpolation/definitions.json#/string_with_placeholders"
-      },
```

Comment:
Title is probably still be needed here as otherwise there is no title for the page.